packages/forma-36-react-components/src/components/Dropdown/README.mdx

```diff
@@ -39,7 +39,7 @@ function DropdownExample() {
         <Button
           size="small"
           variant="transparent"
-          indicateDropdown
+          endIcon={<ChevronDownIcon />}
```

Comment:
I guess we can remove Dropdown component. I think Alex removed it in his PR, but it's not merged yet?